### DISCUSSION TITLE
Copilot chat doesn't appear in my VSCode after I install it

### DISCUSSION THREAD

**Discussion by richard-epsilla** [community/community#60458] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

The intro video shows the new item in the left navigation bar to start copilot chat, but after install it on my VSCode I didn't see it appear
<img width="1182" alt="Screenshot 2023-07-09 at 3 30 14 PM" src="https://github.com/community/community/assets/131846445/6ae675b2-467e-4969-9e73-18eb40837ff0">

**Comment by davecheney** [comment, marked as answer]
@richard-epsilla you are not part of the copilot chat waitlist. Please apply here and wait for a notification. https://github.com/github-copilot/chat_waitlist_signup
